docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,184 @@
+***************************
+Using Ansible with Scaleway
+***************************
+
+Introduction
+============
+
+`Scaleway <https://scaleway.com>`_ is cloud provider supported by Ansible (>=2.6) via a dynamic inventory script and modules.
+Those modules are:
+
+- scaleway_ssh: adds a public SSH key from file or value to the Packet infrastructure. Every subsequently-created device will have this public key installed in .ssh/authorized_keys.
+- scaleway_compute: manages servers on Scaleway. You can use this module to create, restart and delete servers.
+- scaleway_volumes: manages volumes on Scaleway.
+
+Note, this guide assumes you are familiar with Ansible and how it works.
+If you're not, have a look at their :ref:`docs <ansible_documentation>` before getting started.
+
+Requirements
+============
+
+The Scaleway modules and inventory script connect to the Scaleway API using `Scaleway REST API <https://developers.scaleway.com>`_.
+
+In order to check the state of devices created by Ansible on Packet, it's a good idea to install one of the `Packet CLI clients <https://www.packet.net/developers/integrations/api-cli/>`_. Otherwise you can check them via the `Packet portal <https://app.packet.net/portal>`_.
+
+To use the modules and inventory script you'll need a Packet API token. You can generate an API token via the Packet portal `here <https://app.packet.net/portal#/api-keys>`__. The simplest way to authenticate yourself is to set the Packet API token in an environment variable:
```

Comment:
Does Packet offer documentation for using API tokens? These steps are not Ansible-specific, so if documentation exists elsewhere, I'd prefer to link to it rather than duplicate it here.